conda/utils.py

```diff
@@ -286,3 +288,15 @@ def wrap_subprocess_call(on_win, root_prefix, prefix, command):
         command_args = [shell_path, "-x", script_caller]
 
     return script_caller, command_args
+
+
+def ensure_comspec_set():
+    if 'COMSPEC' not in environ:
```

Comment:
@soapy1 we need to revisit this, similar to our usage in the installers.  It's not enough that it is set, it needs to be set to cmd.exe.